Food/go.lua

```diff
@@ -1,5 +1,5 @@
 local name = "go"
-local version = "1.14.7"
+local version = "1.15"
```

Comment:
can you please use  v1.15.0 here?

Reply:
We use the version in the URL too, and official Golang download URLs are not using 1.15.0 - https://golang.org/dl/ . If we change this, we need to change some logic in the code for the URL or use a separate variable maybe where the logic resides. Any particular reason why we want to put 1.15.0 ?

Reply:
Even before we followed not putting 0 for patch version - https://github.com/fishworks/fish-food/pull/594